SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -454,16 +454,23 @@ void CMultizoneDriver::Update() {
     for (auto jZone = 0u; jZone < nZone; jZone++){
       /*--- The target zone is iZone ---*/
       if (jZone != iZone){
+        /*--- Run update on structural solution to update relaxation on grid velocities before transferring data ---*/
+        if (config_container[jZone]->GetStructuralProblem()) {
+          iteration_container[jZone][INST_0]->Update(output_container[jZone], integration_container, geometry_container,
+              solver_container, numerics_container, config_container,
```

Comment:
Would running the Update for all zones before the transfer work?
Having exceptions for some types of problems does not look so nice IMO. And like this there is a change the update of j zone will be called for multiple i zones.

Reply:
When doing the initial implementation, I needed to separate the structural problem as above to update the structural solutions correctly when using relaxation. I've tested with running the update for both zones before the transfer which seems to work fine as well, so I've moved the Update call for all zones.